test/unit/host_aspect_configuration_test.rb

```diff
@@ -0,0 +1,47 @@
+require 'test_helper'
+
+class HostAspectConfigurationTest < ActiveSupport::TestCase
+  class MyCoolClass
```

Comment:
Bikeshedding here, but I'd `s/MyCool/AspectTest/g` `s/my_cool/aspect_test` this file to make the intent clear